src/frontend/src/components/EventDetails.vue

```diff
@@ -12,16 +12,19 @@ import EditCarButton from './EditCarButton.vue';
       <div>
         <h3 class="card-title">{{ event?.name }}</h3>
         <h5 class="card-text">
-          <a target="_blank" v-bind:href="'https://maps.google.com/?q=' + event?.location"
-            ><IconPin /> {{ event?.location }}</a
-          >
+          <a target="_blank" v-bind:href="'https://maps.google.com/?q=' + event?.location">
+            <IconPin /> {{ event?.location }}
+          </a>
         </h5>
         <div v-if="screenStore.mobile">
           <h5><b>Start: </b>{{ startTime }}</h5>
           <h5><b>End: </b>{{ endTime }}</h5>
         </div>
         <h5 class="card-text" v-else>{{ startTime }} - {{ endTime }}</h5>
-        <NeedsRideButton />
+        <div class="d-flex flex-wrap gap-2 pb-2">
+          <NeedsRideButton />
+          <ShareEventButton :event-id="id" />
+        </div>
```

Comment:
Nit: IMO the share button would be cleaner as a little link icon next to the title. But I don't use this app nearly as much as you, so do what you feel is best :)